extensions/src/ACRE2Core/RadioEffectCVSD.h

```diff
@@ -20,10 +21,11 @@ class CRadioEffectCVSD : public CSoundMonoEffect {
     private:
         std::vector<bool> lookback_register;
         const std::size_t lookback = 3;
-        const short delta_min = 32;
-        const short delta_max = delta_min * 16;
-        const short delta_coef = 2;
-        const float decay = 0.95;
+        const short delta_min = 256;
```

Comment:
My first  comment on this. Can you use fixed size integers (for example std::int16_t)?

Reply:
I've done this using shorts as this is what the TS API uses for audio samples (which is a little odd). It's what all the existing effects use also.